### ISSUE TITLE
Change the wording of advice about VPNs

### ISSUE THREAD

**Issue by snim2** [dxw/playbook#999]
Currently we say:

> Avoiding connecting devices to untrustworthy networks (internet cafes, security conferences, unencrypted (open) WiFi networks, etc)

However, we don't recommend the use of the VPN for general use - only for specific access needs. Anyone who does need VPN access (for example to access an IP-restricted  website) needs to make a support request which should include the serial number of the device they will be using with the VPN ("About this Mac").

**Comment by leedxw** [comment]
It doesn't currently say anything about the use of VPNs.

I'm not sure "avoid  untrustworthy networks" is actionable advice. If people are going to work from a coffee shop, they should feel that they can use the available wifi.

This would be overridden by any client-specific policies I imagine, but most of the circa-2010 concerns about wifi networks aren't a practical concern today and shouldn't be hampering people's ability to work remotely.

**Comment by sbrody** [comment]
I agree - unless we have a blanket policy of only working at home or in the office - we need more information about how to make a judgement about whether a network is unsafe. Is using your personal mobile phone's wifi cloud ok? Are there particular measures you should take when using a public network?
